packages/caliper-cli/caliper.js

```diff
@@ -16,13 +16,15 @@
 'use strict';
 
 process.env.SUPPRESS_NO_CONFIG_WARNING = true;
+const { CaliperUtils } = require('@hyperledger/caliper-core');
+const Logger = CaliperUtils.getLogger('@hyperledger/caliper-cli');
```

Comment:
I'd use a simpler name for the logger, something like `CLI`, so it resembles a module name, rather than a package name.